src/main/java/jenkins/branch/OrganizationFolder.java

```diff
@@ -1385,7 +1385,8 @@ public void complete() throws IllegalStateException, IOException, InterruptedExc
                                 } finally {
                                     bc.commit();
                                 }
-                                if (isBuildable()) {
+                                existing.fireSCMSourceAfterSave(existing.getSCMSources());
+                                if (isBuildable() && existing.isBuildable() && existing.updateDigests()) {
```

Comment:
NOTE: we should have been considering the MBP's `isBuildable()` once that method was added.

Reply:
NOTE: we now may suppress excessive indexing of the MBP if the digest is unchanged. Some people may view this as a regression if they were relying on the side-effect of a full scan of an `OrganizationFolder` triggering a full scan of all MBPs... but we also have users who were complaining... and there really is no reason to force a scan of the child MBPs (in fact we have users complaining about unnecessary child scanning)